docker/test_env/ethereum_env.go

```diff
@@ -354,21 +372,10 @@ func (en *EthereumNetwork) startPos() (blockchain.EVMNetwork, RpcProvider, error
 	}
 
 	en.DockerNetworkNames = networkNames
-	//TODO when we support multiple participants, we need to modify net so that it contains all the RPC URLs, not just the last one
+	net.ChainID = int64(en.ethereumChainConfig.ChainID)
+	net.SupportsEIP1559 = true
```

Comment:
So all nets do support EIP1559 by default, except Besu? We turn it on in `ethereum_env.go` by default but I don't see where we set this flag off for Besu. If we skip executing `eth_maxPriorityFeePerGas` if client type is `Besu` and never set this flag to `false` maybe we can remove it?

Reply:
Besu also supports EIP1559, but not our implementation of it's handling, i.e. it works with MetaMask, so I guess they handle a situation, when that very method is not implemented.

With the small change to `blockchain/ethereum.go` that I did in this PR we also handle that and trx are correctly gas-priced and processed.